pkg/cniipwrapper/ip.go

```diff
@@ -47,6 +47,11 @@ func (*cniIP) SetHWAddrByIP(ifName string, ip4 net.IP, ip6 net.IP) error {
 	return ip.SetHWAddrByIP(ifName, ip4, ip6)
 }
 
-func (*cniIP) DelLinkByNameAddr(ifName string, family int) (*net.IPNet, error) {
-	return ip.DelLinkByNameAddr(ifName, family)
+func (*cniIP) DelLinkByNameAddr(ifName string) (*net.IPNet, error) {
```

Comment:
It doesn't seem like we make use of the IP address return (in `deletelink.go`). Maybe we can just return the error.